Asgardia.Constitution.Proposal.txt

```diff
@@ -82,7 +82,7 @@ Article 5. Territory
 § 1. At the time of its foundation, Asgardia’s territory is the digitised noosphere, a digital state with living citizens on Earth, which exists on a near-Earth orbit, in the form of a single satellite or an orbital satellite constellation.

```

Comment:
I see that you guys are switching to UK English (sorry for trying to change the text back to US English :) ). However, there were quite a few words throughout the document that still had US English in use. I have corrected these to UK English